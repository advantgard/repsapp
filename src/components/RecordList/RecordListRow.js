import React from "react";

export const RecordListRow = (props) => {

	return (
		<li className="record-list__row">
			<div className="record-list__label-item"> {props.label} </div>
			<div className="record-list__percentage-item"> {props.percentage} </div>
		</li>
	);

};