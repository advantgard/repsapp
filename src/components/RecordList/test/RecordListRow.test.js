import React from "react";
import renderer from "react-test-renderer";
import { RecordListRow } from "../RecordListRow";

describe( "RecordListRow", () => {

	it( "renders correctly", () => {

		const tree = renderer.create(
			<RecordListRow label={10} percentage={20}/>
		).toJSON();

		expect( tree ).toMatchSnapshot();

	} );

} );