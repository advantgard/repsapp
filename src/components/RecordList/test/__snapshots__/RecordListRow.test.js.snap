// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordListRow renders correctly 1`] = `
<li
  className="record-list__row"
>
  <div
    className="record-list__label-item"
  >
     
    10%
     
  </div>
  <div
    className="record-list__percentage-item"
  >
     
    20 lbs
     
  </div>
</li>
`;
